clang/test/SemaSYCL/unnamed-kernel.cpp

```diff
@@ -32,27 +32,27 @@ struct MyWrapper {
 public:
   void test() {
     cl::sycl::queue q;
-    // expected-error@#KernelSingleTask {{'InvalidKernelName1' should be globally visible}}
+    // expected-error@#KernelSingleTask {{non-forward-declarable type 'InvalidKernelName1' is invalid; provide a forward declarable kernel name at namespace scope}}
```

Comment:
NIT: maybe it is just me but I don't find this diagnostic (as well as SYCL 2020 statements) particularly clear. It likely means that the kernel name should not be declared inside class or function scope (for example), but the first question coming to my mind when I see this message is "at which namespace scope?".
How about something like
"`InvalidKernelName1` is an invalid kernel name; kernel name should be forward declarable at namespace scope`" ?

Reply:
Thanks @Fznamznon for the reviews. I have updated diagnostic message like:

``InvalidKernelName1 is an invalid; kernel name should be forward declarable at namespace scope"